pkg/clusterversion/cockroach_versions.go

```diff
@@ -458,6 +465,21 @@ var versionsSingleton = keyedVersions{
 		Key:     DateAndIntervalStyle,
 		Version: roachpb.Version{Major: 21, Minor: 1, Internal: 1160},
 	},
+	{
+		// V21_2 is CockroachDB v21.2. It's used for all v21.2.x patch releases.
+		Key:     V21_2,
+		Version: roachpb.Version{Major: 21, Minor: 2},
+	},
+
+	// v21.2 versions.
+	// TODO(celia) - should `Internal` be 1160+2? Or should a use something bigger like 2000?
+	{
+		Key:     Start21_2,
```

Comment:
I think we want to use `Start22_1` here;  Start21_2  is already defined above. Also, the version should be `roachpb.Version{Major: 21, Minor: 2, Internal: 2}`. The +14, +100, and +1000 were due to historical artifacts. Pretty much any number >= 2 would work here, but I think we should start at 2.